src/ProjectOrigin.WalletSystem.IntegrationTests/ApiTests/ApiTests.can_create_wallet_endpoint.verified.txt

```diff
@@ -0,0 +1,7 @@
+﻿{
+  walletReference: {
+    version: 1,
+    endpoint: http://localhost/v1/slices,
```

Comment:
Shouldn't there be pathbase on this as well?

Reply:
basepath for this file is empty, the other added test file shows that it is included when set.